idaplugin/rematch/network.py

```diff
@@ -111,15 +119,18 @@ def query(method, url, server=None, token=None, params=None, json=False):
 def get_server(server):
   """getting and finalzing server address"""
 
-  if not server:
-    if 'server' not in config or not config['server']:
+  try:
+   if not server:
+    if 'server' not in config and not config['login']['server']:
       raise exceptions.QueryException()
-    server = config['server']
-  if not (server.startswith("http://") or server.startswith("http://")):
-    server = "http://" + server
-  if not server.endswith("/"):
-    server = server + "/"
-
+    server = config['login']['server']
+   if not (server.startswith("http://") or server.startswith("http://")):
+     server = "http://" + server
+   if not server.endswith("/"):
+     server = server + "/"
+  except:
+    import traceback
+    traceback.print_exc()
```

Comment:
I really don't like all those try/except/print. Lets fix any issues. if for debug, we'll find something more robust.